src/utils/interfaces/counter.ts

```diff
@@ -0,0 +1,4 @@
+export interface IMultipleOfFive {
+    message: 'This number is a multiple of 5';
+    showMessage: boolean;
+}
```

Comment:
All files must end with an empty EOF

Reply:
I meant `All files must end with an empty **line before** EOF.
EOF = End of File, it's a standard practice that all source code files end with an empty new line with the exception of `.json` files.